src/main/java/mate/academy/bookshop/dto/shoppingcart/CartItemRequestDto.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.bookshop.dto.shoppingcart;
+
+public record CartItemRequestDto(
+        Long bookId,
+        int quantity
+) {
```

Comment:
Can we add validation for that dto? because we can receive null book id and 0 quantity now